packages/core/src/domain/internalMonitoring/internalMonitoring.spec.ts

```diff
@@ -242,6 +249,31 @@ describe('internal monitoring', () => {
     let internalMonitoring: InternalMonitoring
     let notifySpy: jasmine.Spy<(event: TelemetryEvent & Context) => void>
 
+    describe('rollout', () => {
+      ;[
+        { site: INTAKE_SITE_US5, enabled: false },
+        { site: INTAKE_SITE_US3, enabled: false },
+        { site: INTAKE_SITE_EU, enabled: false },
+        { site: INTAKE_SITE_US, enabled: false },
+      ].forEach(({ site, enabled }) => {
+        it('should be enabled by site', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏to see which test is failing:
```suggestion
        it(`should be ${enabled ? 'enabled' : 'disabled'} on site ${site}`, () => {
```